deploy/eck-kibana/values.yaml

```
@@ -0,0 +1,44 @@
+---
+# Default values for eck-kibana.
+# This is a YAML-formatted file.
+
+# Overridable names of the Kibana instance.
+# By default, this is the Release name set for the chart,
+# followed by 'eck-kibana'.
+#
+# nameOverride will override the name of the Chart with the name set here,
+# so nameOverride: testing, would convert to '{{ Release.name }}-testing'
+#
+# nameOverride: "quickstart"
+#
+# fullnameOverride will override both the release name, and the chart name,
+# and will name the Kibana instance exactly what is specified.
+#
+# fullnameOverride: "quickstart"
+
+# Version of Kibana.
+#
+version: 8.2.3
+
+# Labels that will be applied to Kibana.
+#
+labels: {}
+
+# Annotations that will be applied to Kibana.
+#
+annotations: {}
+
+spec:
+  # Count of Kibana instances to create.
+  #
+  count: 1
+
+  # Reference to ECK-managed Elasticsearch instance.
+  #
+  elasticsearchRef:
+    name: quickstart
+    # Optional namespace reference to Elasticsearch instance.
+    # If not specified, then the namespace of the Kibana instance
```

```suggestion
    # If not specified, then the namespace of the Kibana resource
```
and so in other places.